presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHive.java

```diff
@@ -2335,9 +2337,17 @@ private void doTestBucketSortedTables(SchemaTableName table)
                 sink.appendPage(builder.build().toPage());
             }
 
-            // verify we have enough temporary files per bucket to require multiple passes
-            Path stagingPathRoot = getStagingPathRoot(outputHandle);
             HdfsContext context = new HdfsContext(session, table.getSchemaName(), table.getTableName());
+            // verify we have enough temporary files per bucket to require multiple passes
+            Path stagingPathRoot;
+            if (isTemporaryStagingDirectoryEnabled(session)) {
+                stagingPathRoot = new Path(getTemporaryStagingDirectoryPath(session)
```

Comment:
don't you need to use `io.prestosql.plugin.hive.util.HiveWriteUtils#createTemporaryPath`?

Reply:
`metadata.beginCreateTable` (a few lines above) calls `locationService.forNewTable`, this has a call to `createTemporaryPath`. Here we are just setting up the right path for listing staged files for verification.